modules/aad/domain-service/README.md

```diff
@@ -212,7 +180,7 @@ $pfxCertificate = [System.Convert]::ToBase64String($rawCertByteStream)
 
 _None_
 
-## Deployment examples
+## Usage Examples
```

Comment:
Let's keep `Deployment examples` for now. We have to do a bulk update before running the ReadMe utility (and move that section because it is in the wrong place. If we were to do this now in main, the changes would break the pipeline.

Reply:
Reverted